### ISSUE TITLE
Feature request - Calculate Public Transport Isochrones

### ISSUE THREAD

**Issue by Leinatan** [riccardoklinger/Hqgis#36]
Hi,

It would be great if it were possible to use the API to calculate public transport isochrones directly in QGIS!

https://developer.here.com/documentation/transit/topics/resource-isochrone-search.html

I am unfortunately totally unable to contribute to any coding, but let me know if I can help in any other way....

**Comment by riccardoklinger** [comment]
Yes this makes sense. It is a bit strange to have a seprate endpoint for those public transit option but it would definitely make sense. 
the endpoint description is in the link provided above. Currently I was just using the route API.

**Comment by Leinatan** [comment]
Hey, 

More than a year later, still needing this! Still not a programmer, but if it's doable with basic coding knowledge, I'm willing to help.

**Comment by Leinatan** [comment]
Ok so apparently PT API v8 doesn't do PT isochrones anymore... https://developer.here.com/documentation/public-transit/dev_guide/index.html

So that's that.
